R/src-vertica.R

```diff
@@ -144,6 +144,16 @@ Vertica.Query <- R6::R6Class("Vertica.Query",
     fetch = function(n = -1L) {
       if(self$con@type == "ODBC") {
         out <- sqlQuery(self$con@conn, self$sql, n)
+	if(class(out) == "character")
+	{
+		errors = grepl("Exception in processPartitionForR", out)
```

Comment:
The R-UDX catches errors from the R code in the UDX so it will catch all the errors that happened on vertica side (assuming the udx actually starts running)